spec/commands/v2/reallocating_base_path_spec.rb

```diff
@@ -0,0 +1,317 @@
+require "rails_helper"
+
+RSpec.describe "Reallocating base paths of content items" do
+  let(:content_id) { SecureRandom.uuid }
+  let(:base_path) { "/vat-rates" }
+  let(:command) { Commands::V2::PutContent }
+
+  before do
+    stub_request(:put, %r{.*draft-content-store.*/content/.*})
+  end
+
+  let(:regular_payload) do
+    FactoryGirl.build(:draft_content_item,
+      content_id: content_id,
+      base_path: base_path
+    ).as_json.deep_symbolize_keys
+  end
+
+  let(:substitute_payload) do
+    FactoryGirl.build(:redirect_draft_content_item,
+      content_id: content_id,
+      base_path: base_path
+    ).as_json.deep_symbolize_keys
+  end
+
+  context "when a base path is reserved by a not-yet-published regular content item" do
```

Comment:
Talking about reserving base paths is a bit misleading because that implies the PathReservation stuff. Maybe "when a not-yet-published regular content item exists at a base path".